graph/quadwriter.go

```diff
@@ -69,11 +69,18 @@ var (
 )
 
 type QuadWriter interface {
+	quad.Writer
+	quad.BatchWriter
+
 	// Add a quad to the store.
+	//
+	// Deprecated: use WriteQuad instead.
```

Comment:
If this name change is going to happen. Can we just delete the old methods. The longer something lies around, the greater the chance someone will use the deprecated method. Except where bound by backwards compatibility guarantees, deprecation is not a normal Go idiom.

\cc @barakmich